tests/common.go

```diff
@@ -985,11 +985,36 @@ func ScheduleApplications(testname string, errChan ...*chan error) []*scheduler.
 	var err error
 
 	Step("schedule applications", func() {
-		taskName := fmt.Sprintf("%s-%v", testname, Inst().InstanceID)
-		contexts, err = Inst().S.Schedule(taskName, scheduler.ScheduleOptions{
+		options := scheduler.ScheduleOptions{
 			AppKeys:            Inst().AppList,
 			StorageProvisioner: Inst().Provisioner,
-		})
+		}
+		//if not hyper converged set up deploy apps only on storageless nodes
+		if !Inst().IsHyperConverged {
+			logrus.Infof("Scheduling apps only on storageless nodes")
+			storagelessNodes := node.GetStorageLessNodes()
```

Comment:
Add a check for a cluster where no storage-less node is available. Code should check and take the appropriate action !!

Reply:
In current logic if there are no storageless nodes, apps will deployed as if hyperconverged is Yes.
I can add a statement that no storageless nodes found deploying apps accross all nodes.
Is that ok?

Reply:
Added the comment